DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesViewModelTests.cs

```diff
@@ -68,7 +64,7 @@ public void GroupDelegatesViewModel_should_correctly_return_the_second_page_of_d
                 model.GroupId.Should().Be(1);
                 model.NavViewModel.Should().BeEquivalentTo(expectedNavViewModel);
                 model.GroupDelegates.Count().Should().Be(5);
-                model.GroupDelegates.First().Name.Should().BeEquivalentTo("k Surname");
+                model.GroupDelegates.First().Name.Should().BeEquivalentTo("K Surname");

```

Comment:
So, yeah, this is still assuming that num delegates on a page is 10. Probably best is to `Skip()` the first page worth and check the test output against the first of the remaining delegates. As much as I like expected values to be literals, I think this is more important. What do you think?